docs/source/conceptual/tensor_parallelism.md

```diff
@@ -0,0 +1,15 @@
+# Tensor Parallelism
+
+Tensor parallelism is a technique used to fit a large model in multiple GPUs. For example, when multiplying the input tensors with the first weight tensor, multiplying both tensors is equivalent to splitting the weight tensor column-wise, multiplying each column with input separately, and then concatenating the separate outputs. These outputs are then sent between GPUs and then concatenated together to get the final result, like below 👇 
+
+![Image courtesy of Anton Lozkhov](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/tgi/TP.png)
```

Comment:
I think the image is slightly misaligned. The bottom part seems more to the right

Reply:
I edited it to look better, but github preview will not reflect that.

Reply:
Omar is right, the bottom box is ~10 pixels more to the right (just downloaded from `documentation-images`)

Reply:
I fixed it.